front/app/containers/Admin/dashboard/overview/charts/SelectableResourceChart.tsx

```diff
@@ -148,10 +174,27 @@ const SelectableResourceChart = ({
   ...reportExportMenuProps
 }: Props & WrappedComponentProps) => {
   const currentChart = useRef();
+  const containerRef = useRef<HTMLDivElement | null>(null);
+  const [showMore, setShowMore] = useState<boolean | null>(null);
 
   const unitName = formatMessage(RESOURCE_MESSAGES[currentSelectedResource]);
   const xlsxEndpoint = XLSX_ENDPOINTS_MAP[currentSelectedResource + byWhat];
 
```

Comment:
Works well! I guess for now we only need this for these type of components, so it's fine to keep it here, but perhaps in the future we might want to extend this to the `GraphCard` more broadly